multi-node/generic/worker-install.sh

```diff
@@ -262,8 +306,11 @@ systemctl stop update-engine; systemctl mask update-engine
 systemctl daemon-reload
 systemctl enable flanneld; systemctl start flanneld
 systemctl enable kubelet; systemctl start kubelet
+
+if [ $CONTAINER_RUNTIME = "rkt" ]; then
+        systemctl enable rkt-api; systemctl start rkt-api
```

Comment:
Yup, I'll put the `RequiredBy`/`Before` stuff in both `rkt-api.service` and `load-rkt-stage1.service` for all files and take out the `kubelet.service` Require` and `After` stuff